configure-bigquery/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Configure BigQuery
```

Comment:
Actually, this one isn't currently being used (it's commented out), so we can probably hold off on this action until QA standards finishes looking into it. I doubt this is the case, but they could end up changing something here